### ISSUE TITLE
Balance Game Play

### ISSUE THREAD

**Issue by ChromesDuzez** [PlatFormPlayZ/CIS-350-Project-1#1]
The random Trash spawner is good except for that we dont have a clever way to get valid spawning spots are for a given environment. What we have would work for a demo but I suspect once we have our first environment it wont be optimal if we want to have more than one map where the game takes place.

**Comment by ChromesDuzez** [comment]
John considering you implemented the script I would imagine you prolly already thought of the aforementioned issue. Do you wanna implement it?

**Comment by MYTEE-MITE** [comment]
I'm not too sure what you mean by this - the trash spawner is a prefab game object that can be added/moved/removed anywhere we want it to be. Multiple can exist and they can be slid put anywhere we want on the map. Are you referring to the fact that they only spawn trash on a 2D plane and don't factor in terrain shape?

**Comment by ChromesDuzez** [comment]
Yes I just wasnt sure how we wanted to handle the spawning for multiple scenarios:
1) not a square/rectangular area of play such as a level that has a circular or an area of play in the shape of an L for exampel where there would be areas that are out of bounds that trash shouldnt be able to spawn on...
2) I think having multiple spawners in a single environment could cause issues with having too much stuff spawning in a given level

More just curious on the logistics of this if you think the current spawner object and having multiple wont cause issues, Imma leave that in ur hands.

**Comment by ChromesDuzez** [comment]
- [x] Balance the trash spawning mechanics
- [x] Balance the amount of time allotted to the player
- [ ] Add the ability to hold more than 1 item.
- [x] Make more ghosts spawn as the instability rises
- [ ] Make ghosts disappear if the instability goes down?
